daft/dataframe/dataframe.py

```diff
@@ -1441,9 +1441,14 @@ def summarize(self) -> "DataFrame":
         return DataFrame(builder)
 
     @DataframePublicAPI
-    def distinct(self) -> "DataFrame":
+    def distinct(self, *on: ColumnInputType) -> "DataFrame":
```

Comment:
Can I make this only column names? It seems generally preferred to allow arbitrary expressions, but that just makes the execution more complex and other DataFrame library dont normally allow it